onnxruntime/core/optimizer/conv_add_act_fusion.cc

```diff
@@ -207,14 +209,34 @@ using NTO = NodesToOptimize;
 
 class FuseConvAddActivationAction : public ReplaceWithNew {
  public:
-  FuseConvAddActivationAction() = default;
+  FuseConvAddActivationAction(const std::string& domain = kMSDomain) : ReplaceWithNew(), domain_{std::move(domain)} {}
 
  private:
-  std::string OpType(const RuntimeState& runtimeState) const override {
-    return (runtimeState.selected_nodes.Target().OpType() == "Conv") ? "FusedConv" : "NhwcFusedConv";
+  std::string OpType(const RuntimeState& runtime_state) const override {
+    const auto& domain = runtime_state.selected_nodes.Target().Domain();
+    const auto& op_type = runtime_state.selected_nodes.Target().OpType();
+    if (domain == kMSDomain) {
+      if (op_type == "Conv") {
+        return "FusedConv";
+      } else if (op_type == "NhwcConv") {
+        return "NhwcFusedConv";
+      } else {
+        ORT_THROW("Unsupported op type: ", op_type);
+      }
+    } else if (domain == kMSInternalNHWCDomain) {
+      if (op_type != "Conv") {
+        ORT_THROW("Unsupported op type: ", op_type);
+      }
+      return "Conv";
+    } else {
+      ORT_THROW("Unsupported domain: ", domain);
```

Comment:
nit: multiple ORT_THROW statements increase the binary size vs. a single one at the end.